be/src/vec/functions/function_java_udf.cpp

```diff
@@ -93,6 +93,10 @@ Status JavaFunctionCall::prepare(FunctionContext* context,
 
         RETURN_IF_ERROR(SerializeThriftMsg(env, &ctor_params, &ctor_params_bytes));
         jni_ctx->executor = env->NewObject(executor_cl_, executor_ctor_id_, ctor_params_bytes);
+
+        jbyte* pBytes = env->GetByteArrayElements(ctor_params_bytes, nullptr);
+        env->ReleaseByteArrayElements(ctor_params_bytes, pBytes, JNI_ABORT);
+        env->DeleteLocalRef(ctor_params_bytes);
```

Comment:
have checked udaf ctor new object as same way,  would you add delete ref together